src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -203,6 +205,11 @@ export class SuggestController implements IEditorContribution {
 			return widget;
 		}));
 
+		// Wire up text overtyping capture
+		this._overtypingCapturer = this._toDispose.add(new IdleValue(() => {
+			return this._toDispose.add(new OvertypingCapturer(this.editor, this.widget.value));
```

Comment:
Passing the widget-value defeats it being  an `IdleValue` - since getting the `value` will instantiate it. My understanding is that you want to pause overtype-recording once IntelliSense is active, right? Didn't test that but you could use these model-event: `onDidTrigger` or `onDidSuggest` for pause and `onDidCancel` for resume